src/slacko.mli

```diff
@@ -860,3 +860,8 @@ val users_set_active: token -> [ `Success | parsed_auth_error | bot_error ] Lwt.
 
 (** Manually sets user presence. *)
 val users_set_presence: token -> presence -> [ `Success | parsed_auth_error | presence_error ] Lwt.t
+
+(**/**)
+(** Sets the base URL to make API requests to. For use in tests. *)
+val set_base_url : string -> unit
```

Comment:
With an inner module, we might not even need to have a function like this. Though ocamldoc probably will output documentation for the unapplied functor, which regular users should not need to care about.